protocol/cosi.go

```diff
@@ -120,12 +122,12 @@ func NewProtocolCosi(node *sda.TreeNodeInstance) (sda.ProtocolInstance, error) {
 
 // Start will call the announcement function of its inner Round structure. It
 // will pass nil as *in* message.
-func (pc *ProtocolCosi) Start() error {
+func (pc *CoSi) Start() error {
```

Comment:
Now that we followed `lint`ing, change `pc` to `c` here and in all other methods.